libs/common-services/src/main/java/uk/gov/di/ipv/core/library/service/CimitUtilityService.java

```diff
@@ -22,18 +25,63 @@ public CimitUtilityService(ConfigService configService) {
         this.configService = configService;
     }
 
+    public int getContraIndicatorScore(List<ContraIndicator> contraIndicators)
```

Comment:
These methods were only used in `CimitUtilityService` anyway, so it makes sense to move them here and keep the CIs and pure data classes.